.github/workflows/continuous-integration-windows.yml

```
@@ -31,8 +31,8 @@ jobs:
         cmake --build build --parallel 4 --config Release
   windows-serial:
     # Serial build on Windows
-    name: Serial VS-2019
-    runs-on: windows-2019
+    name: Serial VS-2025
```

The name is misleading since as you pointed out it is using Visual Studio 2022 which is version 17
```suggestion
    name: Serial Visual Studio 17
```
or 2022 but not 2025

---

Used 2022 as the other build uses years not versions... if anyone needs more info on name and versions, have a look [here](https://learn.microsoft.com/en-us/cpp/overview/compiler-versions?view=msvc-170)